docs/platforms/bebop.rst

```diff
@@ -25,15 +25,30 @@ Installing libEnsemble and Dependencies
 ---------------------------------------
 
 You should have an indication that the virtual environment is activated.
-Install mpi4py_ and libEnsemble in this environment, making sure to reference
+Start by installing mpi4py_ in this environment, making sure to reference
 the preinstalled Intel MPI compiler. Your prompt should be similar to the
 following block:
 
 .. code-block:: console
 
     (my_env) user@login:~$ CC=mpiicc MPICC=mpiicc pip install mpi4py --no-binary mpi4py
+
+libEnsemble can then be installed via ``pip`` or ``conda``. To install via ``pip``:
+
+.. code-block:: console
+
     (my_env) user@login:~$ pip install libensemble
 
+Installing libEnsemble via ``conda`` may contain platform-specific
+optimizations for dependencies like NumPy. You can also choose which MPI libEnsemble uses:
+
+.. code-block:: console
```

Comment:
I would not recommend mpich/openmpi for bebop etc... These systems should just get the default (no-mpi) version. Note: It's true, if they know what they are doing on an mpich-derivative system, they could point to/copy the system MPI libs and it should work due to MPICH ABI compatibility, but I think thats too advanced/potentially confusing for docs (esp. as LD_LIBRARY_PATH won't work).

Your instructions for MPI variants should go in the standalone installation section (do we still have that? I dont want to clog up the quick start too much, we should also maybe list spack variants there, but at most include a link here.)

Reply:
Thanks for the suggestions. Which platforms do you mean by "bebop etc.", each HPC platform on the docs?

Our only dedicated installation section is part of quickstart / README, and then we have platform-specific lines in each platform guide. We should make an advanced installation section somewhere that includes conda and spack variants.